src/components/Button/Button.scss

```diff
@@ -0,0 +1,67 @@
+.uir-Button {
+    position: relative;
+    border: 0; /* override browser's default */
+    color: rgba(47,40,51,.6);
+    min-width: 50px;
+    height: 36px;
+    padding: 9px 18px;
+    margin: 8px 8px 8px 0;   
+    border-radius: 2px;
+    text-align: center;
+    white-space: nowrap;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    transition: all .3s;
+    font-family: "DINOT", sans-serif;
+    box-sizing: border-box;
+    font-size: 12px;
+    vertical-align: middle;
+    user-select: none;
+    outline: none;
+    background-color: rgba(47, 40, 51, .05);
+}
+
+.uir-Button-clear {
+    background: transparent;
+    color: black;
+}
+
+.uir-Button:hover {
+    background-color: rgba(47, 40, 51, .1);
+    color: rgb(47,40,51);
```

Comment:
Also colourisation variables/functions:

```sass
$grey: rgb(47, 40, 51);
...
&:hover {
	background-color: transparentize($grey, .1);
	color: $grey
}
```

Reply:
I prefer `rgba` for this reason. transparentize is a false-friend. These 2 are equivalent:

`
transparentize(#FFF, .9);
rgba(#FFF, .1);
`